pages/common/git-verify-commit.md

```diff
@@ -0,0 +1,17 @@
+# git verify-commit
+
+> Check for GPG verification of commits.
+> If GPG verification isn't present, nothing will be returned, even if extra options are chosen.
```

Comment:
I think this more succinct version is better:
```suggestion
> If no commits are verified, nothing is printed.
```

Reply:
Completely disagree with this suggestion

Reply:
Why?

Reply:
Then how about
```suggestion
> If no commits are verified, nothing will be printed, regardless of options specified.
```